templates/swagger/v1_json.tmpl

```diff
@@ -9115,6 +9243,44 @@
       },
       "x-go-package": "code.gitea.io/gitea/modules/structs"
     },
+    "ProtectBranchForm": {
```

Comment:
It's a bit problematic to reuse the same structure for API from documentation point of view. How do I know the format of XWhiteListTeams and XWhiteListUsers? Preferably they should be arrays. At least they should change name to XWhiteListTeamIDs and XWhiteListUserIDs